server/src/main/java/nl/inl/blacklab/server/requesthandlers/RequestHandlerHits.java

```diff
@@ -193,6 +200,8 @@ public int handle(DataStream ds) throws BlsException {
         ds.startEntry("summary").startMap();
 
         long totalTime = job.threwException() ? -1 : job.timeUserWaited();
+        int numDocs = searchParam.getNumberOfDocs();
+        logger.info("For rule:{}. Total execution time is:{} ms and docs:{}", ruleId, totalTime, numDocs);
```

Comment:
So this is at the info level, but the `ConsoleLogDatabase` always puts everything at the trace level. Doesn't that mean we can only get this line if we log literally everything? That seems kinda painful. If BL is being aggressive about using the info level it might be simplest for us to just add things at the warning/error level and have `ConsoleLogDatabase` respect the passed level.

Reply:
we can only get this line if we log literally everything? => Nop this gets logged at info or above.
BL is using info level logging just fine.

Reply:
👍